servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/importPostfinanceCSV.js

```diff
@@ -168,5 +168,7 @@ num payments successfull: ${numPaymentsSuccessful}
     await transaction.transactionRollback()
     console.info('transaction rollback', { req: req._log(), args, error: e })
     throw e
+  } finally {
+    await refreshAllPots()
```

Comment:
As in servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/rematchPayments.js, it might be worth considering to swallow (or just log) errors here and put `refreshAllPots()` into a `try {} catch {}` block of its own. I find this mutations primary purpose is done, transaction committed. An error thrown at this point is of no help to API user.